akka-docs-dev/rst/java/stream-rate.rst

```diff
@@ -141,9 +141,29 @@ Rate transformation
 Understanding conflate
 ----------------------
 
-*TODO*
+When a fast producer can not be informed to slow down by backpressure or some other signal, conflate might be useful to combine elements from a producer until a demand signal comes from a consumer.
+
+Below is an example snippet that summarizes fast stream of elements to a standart deviation, mean and count of elements that have arrived  while the stats have been calculated.
+
+.. includecode:: ../../../akka-samples/akka-docs-java-lambda/src/test/java/docs/stream/RateTransformationDocTest.java#conflate-summarize
+
+This example demonstrates that such flow's rate is decoupled. Element rate at the start of the flow can be much higher that the element rate at the end of the flow.
```

Comment:
[Apostrophe](http://www.grammarbook.com/punctuation/apostro.asp) is used to indicate a possession. :)